src/test/java/org/commcare/formplayer/services/FormSessionsPaginationTest.java

```diff
@@ -52,10 +54,10 @@ public void testPagination() {
         formsSessionsRequestBean.setPageSize(5);
         formsSessionsRequestBean.setPageNumber(3);
         List<FormSessionListView> formSessions = formSessionService.getSessionsForUser("momo", formsSessionsRequestBean);
-        assert formSessions.size() == 3;
+        assertEquals(formSessions.size(), 3);
         for (int i = 0; i < formSessions.size(); i++) {
             // first 3 records in latest first order
-            assert formSessions.get(i).getTitle().contentEquals("More momo " + (2-i));
+            assertEquals(formSessions.get(i).getTitle(), "More momo " + (2-i));
```

Comment:
this failed in the first build so I changed in the hopes of getting better error messages and then it passed 🤷🏻